msteams-platform/resources/schema/manifest-localization.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Localization manifest schema reference
+description: Describes the localization schema supported by the localization manifest for Microsoft Teams
+keywords: teams manifest schema localization
+ms.date: 05/15/2019
+---
+
+# Reference: Localization manifest schema for Microsoft Teams
+
+The Microsoft Teams localization manifest describes language translations that will be served based on the client language settings. Your file must conform to the schema hosted at [`https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.Localization.schema.json`]( https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.Localization.schema.json). Previous versions 1.0-1.4 are also supported (using "v1.x" in the URL). For additional information see [app localization](~/publishing/apps-localization.md).
+
+## Sample
+
+```json
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.Localization.schema.json",
+  "name.short": "Le App Studio",
+  "name.full": "App Studio pour Microsoft Teams",
+  "description.short": "Créez d'excellentes applications pour Microsoft Teams avec App Studio.",
+  "description.full": "Créez de nouvelles applications Microsoft Teams, concevez et prévisualisez des cartes bot, et explorez la documentation avec App Studio.",
+  "staticTabs[0].name": "Editeur de manifest",
+  "staticTabs[1].name": "Editeur de cartes",
+  "staticTabs[2].name": "Bibliothèque de contrôles",
+  "bots[0].commandLists[0].commands[0].title": "chercher",
+  "bots[0].commandLists[0].commands[0].description": "Rechercher la documentation Teams pertinente"
+}
+```
+
+The schema defines the following properties:
+
+## $schema
+
+**URI**
+
+The https:// URL referencing the JSON Schema for the manifest.
+
+> [!TIP]
+> Specify the schema at the beginning of your manifest to enable IntelliSense or similar support from your code editor: `"$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.Localization.schema.json",`
+
+## name.short
```

Comment:
Im not sure if all of this detail is useful. It could be boiled down to, the localixation file is a json file with keys that correspond to json paths of the string to localize in the manifest, the value is the localized string.